src/Database/PG/Query/Connection.hs

```diff
@@ -391,7 +391,8 @@ mkPGRetryPolicy noRetries =
     CR.limitRetriesByDelay limitDelay $
     CR.exponentialBackoff baseDelay <> CR.limitRetries noRetries
```

Comment:
While you're here, i think it might be better to call this `numRetries`; I did a double-take on `noRetries` before I realized what it was supposed to mean.

Reply:
Also, is there a reason we use `exponentialBackoff` rather than [`fullJitterBackoff`](https://hackage.haskell.org/package/retry-0.8.1.2/docs/Control-Retry.html#v:fullJitterBackoff)?

I recognize that this is out-of-scope for your current change, but the only reason (in the past) that I've avoided `fullJitterBackoff` is when I haven't had a `MonadIO` constraint but that doesn't seem to be the case here.

Reply:
Good points. Regarding `fullJitterBackoff`, I don't want to put in the effort to try to understand that now, I'll file a follow-up issue with the suggestion so it's not forgotten.